lk/src/main.rs

```diff
@@ -1,43 +1,34 @@
 mod bash_file;
-mod config;
 mod executables;
-// mod history;
 mod script;
 mod shells;
 mod ui;
 
-use std::path::PathBuf;
-
 use anyhow::Result;
 use bash_file::BashFile;
+use config::{Config, File};
 use executables::Executables;
 use fuzzy_finder::item::Item;
 use fuzzy_finder::FuzzyFinder;
-use log::LevelFilter;
+use log::{info, LevelFilter};
 use log4rs::append::file::FileAppender;
-use log4rs::config::{Appender, Config, Root};
+use log4rs::config::{Appender, Root};
 use log4rs::encode::pattern::PatternEncoder;
-use pastel_colours::{GREEN_FG, RED_FG, RESET_FG};
 use script::Function;
 use shells::UserShell;
 use spinners::{Spinner, Spinners};
+use std::path::{Path, PathBuf};
 use structopt::StructOpt;
 use tempfile::tempdir;
 use ui::{print_bad_function_name, print_bad_script_name};
 
-// use crate::history::History;
-use crate::script::Script;
-
 /// Use lk to explore and execute scripts in your current directory,
 /// and in its sub-directories. lk offers two options: 'list' or 'fuzzy'.
 /// 'list' lets you explore your scripts and their functions in a
 /// hierarchical way. 'fuzzy' lets you do a fuzzy search over all the
 /// scripts and functions found by lk.
 #[derive(StructOpt)]
 struct Cli {
-    /// Set the default mode: fuzzy or list
-    #[structopt(long, short)]
-    default: Option<String>,
```

Comment:
I quite liked being able to set this through the command line, instead of having to go to a file.